test/Unit/Util/ParatestFunctionTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ParaTest\Tests\Unit\Util;
+
+use ParaTest\Util\ParatestFunction;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ *
+ * @covers \ParaTest\Util\ParatestFunction
+ */
+final class ParatestFunctionTest extends TestCase
+{
+    public function testCreateScriptForParatest(): void
+    {
+        self::setArgvParameter();
+        ParatestFunction::createScriptForParatest($_SERVER['argv'], __DIR__);
```

Comment:
1. We can accept the array argument of `createScriptForParatest` by reference, so in this test we never alter the original `$_SERVER['argv']` that could have unexpected issues with PHPUnit
2. Return value is not tested